Universal/brg0-methods/manual.md

```diff
@@ -0,0 +1,51 @@
+# Fixing Property Injection: Manual
```

Comment:
Yeah I did my best with my very limited understanding of ACPI. I figured there'd be some issues with it.
> This would be much more suited for an automated tool tbh
Maybe something to ask Corp about? 👀